README.md

```diff
@@ -1 +1,240 @@
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img/>
+  <br/>
+
+  <h3><b>Read me</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 Rescue the Earth page <a name="about-project"></a>
+
+I built the webpage for Rescue the Earth community,
+using data from some journals and a design of <a href="https://www.behance.net/adagio07">Cindy Shin<a> 
+
+
+## 🛠 Built With <a name="built-with"></a>
+-HTML
+-CSS
+-JS
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The home page displays all the main information about the community as weel as speakers information
+The about page displays information about previous community work
+It has a mobile version too for each page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://onlyunknown.github.io/Capstone-Mudole1/">Live view</a><br>
+<a href="https://www.loom.com/share/9c5d9f4d85cc48f3a8a7a3c053246570">The Recording<a>
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+It is a portfolio project so this could make the user able to see my recent work
+the frameworks that has been used and the program languages has been used.
+
+
+### Prerequisites
+
+Not necessary
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+ cd my-folder
+  git clone git@github.com:OnlyUnknown/Capstone-Mudole1.git
+
+<!--
+Example commands:
+
+```sh
+ 
+```
+--->
+
+### Install
+
+Install this project with:
+
+cd Capstone-Mudole1
+  gem install
+<!--
+Example command:
+
+```sh
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+-VScode 
+-The link of the live version above
```

Comment:
- [ ] Please, I think you can remove these sections because they are not related to this project.👌